gateway-server/src/main/java/org/apache/knox/gateway/services/factory/TokenStateServiceFactory.java

```diff
@@ -61,6 +64,7 @@ protected ServiceType getServiceType() {
 
   @Override
   protected Collection<String> getKnownImplementations() {
-    return unmodifiableList(asList(DefaultTokenStateService.class.getName(), AliasBasedTokenStateService.class.getName(), JournalBasedTokenStateService.class.getName()));
+    return unmodifiableList(asList(DefaultTokenStateService.class.getName(), AliasBasedTokenStateService.class.getName(), JournalBasedTokenStateService.class.getName(),
```

Comment:
This is not necessarily directly related to this PR, but why isn't the ServiceLoader mechanism used for identifying service implementations available on the classpath?

Reply:
We may implement this enhancement in a different JIRA, but I think it's out-of-scope of this one.